integration/spark-common-test/src/test/scala/org/apache/carbondata/integration/spark/testsuite/preaggregate/TestPreAggregateTableSelection.scala

```diff
@@ -324,6 +329,8 @@ test("test PreAggregate table selection with timeseries and normal together") {
     sql("drop table if exists mainTable")
     sql("drop table if exists mainTable_avg")
     sql("drop table if exists lineitem")
+    CarbonProperties.getInstance()
+      .addProperty(CarbonCommonConstants.VALIDATE_DIRECT_QUERY_ON_DATAMAP, "true")
     sql("DROP TABLE IF EXISTS maintabletime")
   }
 
```

Comment:
add a test case to verify user can not query the child table when CarbonCommonConstants.VALIDATE_DIRECT_QUERY_ON_DATAMAP is false